### ISSUE TITLE
Error in help of isHomogeneous

### ISSUE THREAD

**Issue by dadexix86** [Macaulay2/M2#406]
In [the documentation of `isHomogeneous`](http://www.math.uiuc.edu/Macaulay2/doc/Macaulay2-1.6/share/doc/Macaulay2/Macaulay2Doc/html/_is__Homogeneous.html) there is an example that states

> Rings may be graded, with generators having degree 0. For example, in the ring B below, every element of A has degree 0.
> 
> ```
> i4 : A = QQ[a,b,c];
> i5 : B = A[x,y];
> i6 : isHomogeneous B
> o6 = true
> i7 : isHomogeneous ideal(a*x+y,y^3-b*x^2*y)
> o7 = false
> ```

Which is false, because for example
```
i34 : a
o34 = a
o34 : A
i35 : degree a_B
o35 = {0, 1}
o35 : List
```
And indeed, if they had degree 0 the two polynomials were homogeneous
```
i13 : A=QQ[a,b,c,DegreeRank=>0]
o13 = A
o13 : PolynomialRing
i14 : B = A[x,y];
i15 : degree a_B
o15 = {0}
o15 : List
i16 : isHomogeneous ideal(a*x+y,y^3-b*x^2*y)
o16 = true
```

**Comment by DanGrayson** [comment]
Indeed, the documentation is wrong.  What happens is that the ring B bi-graded.

To get the older behavior, one may write
```
B = A[x,y,DegreeMap => x -> {0}, Join => false]
```

**Comment by DanGrayson** [comment]
fixed in commit 44cffd58f57f6630a3eeee48840cc91ce6c99a55
